README.org

```diff
@@ -1,9 +1,18 @@
 #+title: Awesome Unison
 
 A list of Unison related links, libraries, documentation and more.
+
 * Talks
 [[https://www.youtube.com/watch?v=gCWtkvDQ2ZIa][Strange loop]], Unison introduction by Paul Chiusano.
+
+[[https://www.youtube.com/playlist?list=PLQ0IlHfOk1GgbXSZAjOOls9PnrO4Dpsbb][Unison Forall]], all the talks from the Unison Forall 2022 conference.
+
 * Learn
-[[https:unison-lang.org/docs][Official documentation]], language fundementals and more.
 
+[[https://www.unison-lang.org/docs/][Official documentation]], language fundementals and more.
```

Comment:
@hojberg Just curious, any reason for using the `[[url][text]]` syntax for links instead of one of [the other](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links) explicitly supported markdown link methods? I haven't seen this particular format before.

Reply:
@nick-w-nick I use Emacs org-mode which is a "mode" in Emacs which is a well supported markup language that am really comfortable in.

Reply:
Very cool, thanks for the clarification!